BUILD.md

```diff
@@ -0,0 +1,137 @@
+# Hyperledger Cactus Build Instructions
+
+This is the place to start if you want to give Cactus a spin on your local
+machine or if you are planning on contributing.
+
+> This is not a guide for `using` Cactus for your projects that have business logic
+> but rather a guide for people who want to make changes to the code of Cactus.
+> If you are just planning on using Cactus as an npm dependency for your project,
+> then you might not need this guide at all.
+
+The project uses Typescript for both back-end and front-end components.
+
+## Getting Started
+
+* Install OS level dependencies:
```

Comment:
Are you sure all of them are required to build SDK ?
I believe you can just say "we need docker" and provide URL like for further instruction https://docs.docker.com/docker-for-windows/install-windows-home/

I thought OpenJDK is required only when you build Ledger Plugin for Besu. This condition should be mentioned here.

Reply:
> Are you sure all of them are required to build SDK ?
> I believe you can just say "we need docker" and provide URL like for further instruction https://docs.docker.com/docker-for-windows/install-windows-home/

Yes, we don't yet have a build image for docker that would contain all of this, but I agree it's definitely something we should have as an option.

> I thought OpenJDK is required only when you build Ledger Plugin for Besu. This condition should be mentioned here.

It does not, the current besu plugin interacts with the ledger entirely through HTTP so no JDK is necessary for that.
The JDK is needed for the OpenAPI (Swagger) generator which is a part of the build where the declarative API definitions get converted into source code and documentation.